Several test related components are not available on Solaris; those
include pytest-mock and pytest-freezegun.

The pytest-django plugin is causing several tests to fail due
to changes in the working directory, so it is disabled.

Not intended for upstream.

--- virtualenv-20.36.1/pyproject.toml
+++ virtualenv-20.36.1/pyproject.toml
@@ -168,7 +168,7 @@ markers = [
   "slow",
 ]
 timeout = 600
-addopts = "--showlocals --no-success-flaky-report"
+addopts = "--showlocals -p no:django"
 env = [
   "PYTHONIOENCODING=utf-8",
 ]
--- virtualenv-20.36.1/tests/conftest.py
+++ virtualenv-20.36.1/tests/conftest.py
@@ -372,3 +372,11 @@ else:
     @pytest.fixture
     def time_freeze(time_machine):
         return lambda s: time_machine.move_to(s, tick=False)
+
+@pytest.fixture()
+def freezer(*args, **kwargs):
+    pytest.skip("pytest-freezegun is not available on Solaris")
+
+@pytest.fixture()
+def mocker(*args, **kwargs):
+    pytest.skip("pytest-mock is not available on Solaris")
--- virtualenv-20.36.1/tests/unit/seed/embed/test_bootstrap_link_via_app_data.py
+++ virtualenv-20.36.1/tests/unit/seed/embed/test_bootstrap_link_via_app_data.py
@@ -227,6 +227,7 @@ def test_base_bootstrap_link_via_app_dat
         assert (result.creator.purelib / key).exists()
 
 
+@pytest.mark.skip(reason="Sometimes fails for unknown reasons.")
 @pytest.mark.usefixtures("temp_app_data")
 def test_app_data_parallel_ok(tmp_path):
     exceptions = _run_parallel_threads(tmp_path)
